client/extensions/woocommerce/state/sites/settings/email/reducer.js

```diff
@@ -21,8 +26,50 @@ export default createReducer( null, {
 	[ WOOCOMMERCE_EMAIL_SETTINGS_REQUEST_SUCCESS ]: ( state, { data } ) => {
 		const options = {};
 		data.forEach( function( option ) {
-			setWith( options, [ option.group_id, option.id ], option.value, Object );
+			setWith(
+				options,
+				[ option.group_id, option.id ],
+				{
+					value: option.value,
+					default: option.default, // deafualt_value -> default is a reserved word
+				},
+				Object,
+			);
 		} );
 		return options;
 	},
+
+	[ WOOCOMMERCE_EMAIL_SETTINGS_CHANGE ]: ( state, { setting } ) => {
+		if ( ! setting ) {
```

Comment:
do we need to have a null check for `setting.setting` and `setting.option` here too?

Reply:
I don't think so. We could but I think tat this would be too much. If it would be used outside email settings scope then for sure it would be good to have it.

Reply:
Are you convinced that the checks should be there?

Reply:
It seems like it would be unlikely that we hit a null here, but we could add the check just to be safe. I'm not fully convinced either way is the best. But when in doubt, seems like adding in the null check is worth it.